dbms/tests/instructions/developer_instruction_en.md

```diff
@@ -0,0 +1,268 @@
+Building of ClickHouse is supported on Linux, FreeBSD and Mac OS X.
+
+# If you use Windows
+
+If you use Windows, you need to create a virtual machine with Ubuntu. To start working with a virtual machine please install VirtualBox. You can download Ubuntu from the website: https://www.ubuntu.com/#download Please create a virtual machine from the downloaded image (you should reserve at least 4GB of RAM for it). To run a command line terminal in Ubuntu, please locate a program containing the word "terminal" in its name (gnome-terminal, konsole etc.) or just press Ctrl+Alt+T.
+
+
+# Creating a repository on GitHub
+
+To start working with ClickHouse repository you will need a GitHub account.
```

Comment:
`you will need` -> `you need`

Reply:
A matter of style once again and both would be correct. I take it that the original line meant "if you wish to start working ...". English is very flexible about how you indicate time. I choose to use "will" unless something else suggests it's a conditional or future need.